runners/spark/src/main/java/org/apache/beam/runners/spark/translation/SparkContextFactory.java

```diff
@@ -85,6 +86,8 @@ private static JavaSparkContext createSparkContext(SparkPipelineOptions options)
         conf.setMaster(options.getSparkMaster());
       }
       conf.setAppName(options.getAppName());
+      // register immutable collections serializers because the SDK uses them.
```

Comment:
I'm generally pro-testing but isn't it a little bit of an overkill to test Spark's features ? this is a Spark configuration feature which is [tested](https://github.com/apache/spark/blob/branch-1.6/core/src/test/scala/org/apache/spark/serializer/KryoSerializerSuite.scala#L223) by Apache Spark.
The custom serializers are tested by `kryo-serializers`.
Should I invest in testing tested features ?